src/stories/Cover.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import { API, Position, Size } from 'types';
+import { PanZoomWithCover, Element } from '..';
+import styles from './Cover.module.css';
+
+export default {
+  title: 'Cover',
+};
+
+const PINS_INITIAL = {
+  pin1: { x: 313, y: 181 },
+  pin2: { x: 1463, y: 323 },
+};
+const PIN_SIZE = 48; // px
+const PIN_HALF_WIDTH = PIN_SIZE / 2;
+const PIXEL_TO_METER = 3.794;
+const DISTANCE_THROTTLE_TIMEOUT = 200; // ms
+
+const Pin = () => (
+  <svg
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    className={styles.pin}
+  >
+    <path d="M12 10c-1.104 0-2-.896-2-2s.896-2 2-2 2 .896 2 2-.896 2-2 2m0-5c-1.657 0-3 1.343-3 3s1.343 3 3 3 3-1.343 3-3-1.343-3-3-3m-7 2.602c0-3.517 3.271-6.602 7-6.602s7 3.085 7 6.602c0 3.455-2.563 7.543-7 14.527-4.489-7.073-7-11.072-7-14.527m7-7.602c-4.198 0-8 3.403-8 7.602 0 4.198 3.469 9.21 8 16.398 4.531-7.188 8-12.2 8-16.398 0-4.199-3.801-7.602-8-7.602" />
+  </svg>
+);
+
+export const cover = () => {
+  const [distance, setDistance] = useState<number>(null);
+  const [timer, setTimer] = useState<NodeJS.Timeout>(null);
```

Comment:
```suggestion
  const [timer, setTimer] = useState<ReturnType<typeof setTimeout>>(null);
```

Reply:
They're the same aren't them? The return type of setTimeout. Or not?

Reply:
It dependes where we are running this code. if it's on Node (for example running tests) it's okay, but in browser it would be `number`, so I think safest is ReturnType<typeof setTimeout>>

maybe check this https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window

Reply:
Ok thanks. I've always been confused about the return type of setTimeout. Always thought it's number (timer id)